### ISSUE TITLE
G915 TKL and Superlight Wireless Pro Mouse not detecting

### ISSUE THREAD

**Issue by Entharia** [libratbag/libratbag#1594]
**Information**

- `ratbagd` version (`ratbagd --version`):0.17
- Distribution: 
- Kernel version (ex. `uname -srmo`): Linux 6.7.6-arch1-1 x86_64 GNU/Linux

Device info (if applicable):
```
$ ratbagctl <device> info
OUTPUT HERE
```

**Describe the bug**
The command ratbagctl list gives me this output:
No devices available.

But both devices exist as a .device file

**Logs**
Please start the daemon with verbosity and reproduce the issue.

First make sure it isn't running already.
```
$ ratbagd --verbose=raw
OUTPUT HERE
Initializing libratbag
ratbag debug: New device: Logitech USB Receiver
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: device assigned driver hidpp20
ratbag debug: hidraw info: bus 0x03 vendor 0x46d product 0xffffc547
ratbag debug: Logitech USB Receiver is device '/dev/hidraw1'.
ratbag debug: Parsing HID report descriptor
ratbag debug: Parsing HID report descriptor
ratbag debug: hidraw info: bus 0x03 vendor 0x46d product 0xffffc547
ratbag debug: Logitech USB Receiver is device '/dev/hidraw2'.
ratbag debug: Parsing HID report descriptor
ratbag debug: Parsing HID report descriptor
ratbag debug: - HID report ID 01
ratbag debug: - HID report ID 03
ratbag debug: - HID report ID 04
ratbag debug: hidraw info: bus 0x03 vendor 0x46d product 0xffffc547
ratbag debug: Logitech USB Receiver is device '/dev/hidraw3'.
ratbag debug: Parsing HID report descriptor
ratbag debug: Parsing HID report descriptor
ratbag debug: - HID report ID 10
ratbag debug: - HID report ID 11
ratbag debug: hidpp: device supports short reports
ratbag debug: hidpp: device supports long reports
ratbag raw: hidpp write:  10 02 00 18 00 00 00
ratbag raw: hidpp read:   10 02 8f 00 18 08 00
ratbag debug:     HID++ error from the device (2): ERR_BUSY (08)
ratbag debug: New device: Logitech USB Receiver
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: device assigned driver hidpp20
ratbag debug: hidraw info: bus 0x03 vendor 0x46d product 0xffffc547
ratbag debug: Logitech USB Receiver is device '/dev/hidraw1'.
ratbag debug: Parsing HID report descriptor
ratbag debug: Parsing HID report descriptor
ratbag debug: hidraw info: bus 0x03 vendor 0x46d product 0xffffc547
ratbag debug: Logitech USB Receiver is device '/dev/hidraw2'.
ratbag debug: Parsing HID report descriptor
ratbag debug: Parsing HID report descriptor
ratbag debug: - HID report ID 01
ratbag debug: - HID report ID 03
ratbag debug: - HID report ID 04
ratbag debug: hidraw info: bus 0x03 vendor 0x46d product 0xffffc547
ratbag debug: Logitech USB Receiver is device '/dev/hidraw3'.
ratbag debug: Parsing HID report descriptor
ratbag debug: Parsing HID report descriptor
ratbag debug: - HID report ID 10
ratbag debug: - HID report ID 11
ratbag debug: hidpp: device supports short reports
ratbag debug: hidpp: device supports long reports
ratbag raw: hidpp write:  10 02 00 18 00 00 00
ratbag raw: hidpp read:   10 02 8f 00 18 08 00
ratbag debug:     HID++ error from the device (2): ERR_BUSY (08)
ratbag debug: New device: Logitech USB Receiver
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: device assigned driver hidpp20
ratbag debug: hidraw info: bus 0x03 vendor 0x46d product 0xffffc547
ratbag debug: Logitech USB Receiver is device '/dev/hidraw1'.
ratbag debug: Parsing HID report descriptor
ratbag debug: Parsing HID report descriptor
ratbag debug: hidraw info: bus 0x03 vendor 0x46d product 0xffffc547
ratbag debug: Logitech USB Receiver is device '/dev/hidraw2'.
ratbag debug: Parsing HID report descriptor
ratbag debug: Parsing HID report descriptor
ratbag debug: - HID report ID 01
ratbag debug: - HID report ID 03
ratbag debug: - HID report ID 04
ratbag debug: hidraw info: bus 0x03 vendor 0x46d product 0xffffc547
ratbag debug: Logitech USB Receiver is device '/dev/hidraw3'.
ratbag debug: Parsing HID report descriptor
ratbag debug: Parsing HID report descriptor
ratbag debug: - HID report ID 10
ratbag debug: - HID report ID 11
ratbag debug: hidpp: device supports short reports
ratbag debug: hidpp: device supports long reports
ratbag raw: hidpp write:  10 02 00 18 00 00 00
ratbag raw: hidpp read:   10 02 8f 00 18 08 00
ratbag debug:     HID++ error from the device (2): ERR_BUSY (08)
ratbag debug: New device: Kingston HyperX Quadcast
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: No data file found for 0951:16df
ratbag debug: New device: SteelSeries  SteelSeries Arctis 7
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: No data file found for 1038:12ad
ratbag debug: New device: Logitech USB Receiver
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: device assigned driver hidpp20
ratbag debug: hidraw info: bus 0x03 vendor 0x46d product 0xffffc547
ratbag debug: Logitech USB Receiver is device '/dev/hidraw4'.
ratbag debug: Parsing HID report descriptor
ratbag debug: Parsing HID report descriptor
ratbag debug: hidraw info: bus 0x03 vendor 0x46d product 0xffffc547
ratbag debug: Logitech USB Receiver is device '/dev/hidraw5'.
ratbag debug: Parsing HID report descriptor
ratbag debug: Parsing HID report descriptor
ratbag debug: - HID report ID 01
ratbag debug: - HID report ID 03
ratbag debug: - HID report ID 04
ratbag debug: hidraw info: bus 0x03 vendor 0x46d product 0xffffc547
ratbag debug: Logitech USB Receiver is device '/dev/hidraw6'.
ratbag debug: Parsing HID report descriptor
ratbag debug: Parsing HID report descriptor
ratbag debug: - HID report ID 10
ratbag debug: - HID report ID 11
ratbag debug: hidpp: device supports short reports
ratbag debug: hidpp: device supports long reports
ratbag raw: hidpp write:  10 02 00 18 00 00 00
ratbag raw: hidpp read:   10 02 8f 00 18 08 00
ratbag debug:     HID++ error from the device (2): ERR_BUSY (08)
ratbag debug: New device: Logitech USB Receiver
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: device assigned driver hidpp20
ratbag debug: hidraw info: bus 0x03 vendor 0x46d product 0xffffc547
ratbag debug: Logitech USB Receiver is device '/dev/hidraw4'.
ratbag debug: Parsing HID report descriptor
ratbag debug: Parsing HID report descriptor
ratbag debug: hidraw info: bus 0x03 vendor 0x46d product 0xffffc547
ratbag debug: Logitech USB Receiver is device '/dev/hidraw5'.
ratbag debug: Parsing HID report descriptor
ratbag debug: Parsing HID report descriptor
ratbag debug: - HID report ID 01
ratbag debug: - HID report ID 03
ratbag debug: - HID report ID 04
ratbag debug: hidraw info: bus 0x03 vendor 0x46d product 0xffffc547
ratbag debug: Logitech USB Receiver is device '/dev/hidraw6'.
ratbag debug: Parsing HID report descriptor
ratbag debug: Parsing HID report descriptor
ratbag debug: - HID report ID 10
ratbag debug: - HID report ID 11
ratbag debug: hidpp: device supports short reports
ratbag debug: hidpp: device supports long reports
ratbag raw: hidpp write:  10 02 00 18 00 00 00
ratbag raw: hidpp read:   10 02 8f 00 18 08 00
ratbag debug:     HID++ error from the device (2): ERR_BUSY (08)
ratbag debug: New device: Logitech USB Receiver
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: device assigned driver hidpp20
ratbag debug: hidraw info: bus 0x03 vendor 0x46d product 0xffffc547
ratbag debug: Logitech USB Receiver is device '/dev/hidraw4'.
ratbag debug: Parsing HID report descriptor
ratbag debug: Parsing HID report descriptor
ratbag debug: hidraw info: bus 0x03 vendor 0x46d product 0xffffc547
ratbag debug: Logitech USB Receiver is device '/dev/hidraw5'.
ratbag debug: Parsing HID report descriptor
ratbag debug: Parsing HID report descriptor
ratbag debug: - HID report ID 01
ratbag debug: - HID report ID 03
ratbag debug: - HID report ID 04
ratbag debug: hidraw info: bus 0x03 vendor 0x46d product 0xffffc547
ratbag debug: Logitech USB Receiver is device '/dev/hidraw6'.
ratbag debug: Parsing HID report descriptor
ratbag debug: Parsing HID report descriptor
ratbag debug: - HID report ID 10
ratbag debug: - HID report ID 11
ratbag debug: hidpp: device supports short reports
ratbag debug: hidpp: device supports long reports
ratbag raw: hidpp write:  10 02 00 18 00 00 00
ratbag raw: hidpp read:   10 02 8f 00 18 08 00
ratbag debug:     HID++ error from the device (2): ERR_BUSY (08)
DBus server ready

```

Please help :)

**Comment by staticssleever668** [comment]
Hello! Try connecting devices by wire to setup them with libratbag, the settings should be saved to devices, and then you can connect them wirelessly again.

This is tracked in #1119 and #1178.
TLDR: this depends on Linux, which doesn't fully support their wireless receivers yet.

Also, you can try solaar (https://pwr-solaar.github.io/Solaar/).
